bin/deploy.sh

```diff
@@ -56,6 +66,12 @@ for region in "${DEIS_REGIONS[@]}"; do
     echo "Pulling $DOCKER_IMAGE_TAG into Deis app $appname in $region"
     ./deis pull "$DOCKER_IMAGE_TAG" -a "$appname"
 
+    # us-west has the r/w database
+    if [[ "$region" == "us-west" && elementIn "$appname" "${DEIS_POST_DEPLOY_APPS[@]}" ]]; then
```

Comment:
NIT: instead of a comment and hard coding us-west, you could document it with a variable instead, such as
```sh
RW_REGION=us-west
if [[ "$region" == $RW_REGION && [...]
```
it's a stylistic preference, and the comment might make more sense-- just a suggestion